cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -212,9 +212,8 @@ def create_cephCSIKeyring_cephFSProvisioner(self):
         ret_val, json_out, err_msg = self._common_cmd_json_gen(cmd_json)
```

Comment:
You must not create that key if no Cephfs fs/pool is found. Here it is still being created so `len(json_out) == 0:` will never be 0, unless I missed something...